components/controlled.mdx

```diff
@@ -0,0 +1,16 @@
+If you think the data flow in react carefully.
```

Comment:
What's this file for? Do you want to include this into the doc site or just here for developers to check?